Software/Development/SemesterProject/App/digitalInputManager.c

```diff
@@ -40,9 +42,12 @@ static struct
  */
 void initInputs(void)
```

Comment:
Should this function be inside digitalInputs instead of digitalInputManager? I think it would make more sense in there.

Reply:
Discussed in eprsonm we are decoupling the whole list from every single object.